exception DivByZero;
def main() raises DivByZero {
    print(1);
}
